previous_code/gflownetenv.py

```diff
@@ -88,7 +93,7 @@ def reward_batch(self, states, done):
         reward = np.zeros(len(done))
         reward[list(done)] = self.proxy2reward(self.proxy(self.state2oracle(states)))
         return reward
-
+    #Idea TODO : we could make this acq2reward function specific to each acquisition function ! it would be a property method of each acquisition function class
```

Comment:
Right makes sense. That would involve moving the `acq2reward()` to the acquisition class from the `env` class if I understood correctly

Reply:
Not for now, it is quite unclear if acq2rewards is environment specific to tune or acquisition function specific. It could be both acquisition function specific and environment specific, which would be a mess to implement in a modular way ...